web/src/main/java/org/phoenixctms/ctsms/web/jersey/compare/ResourceClassComparator.java

```diff
@@ -4,10 +4,11 @@
 
 import org.phoenixctms.ctsms.compare.AlphanumComparatorBase;
 
-public class ResourceClassComparator implements Comparator<Class> {
+public class ResourceClassComparator implements Comparator<Class<?>> {
```

Comment:
either for all places/projects, or none.

Reply:
There are many raw-reference occurrences, but i had to start somewhere.

Reply:
i presume it is about warnings, or is it a showstopper for new jersey/swagger?

Reply:
not a showstopper but less warnings.